README.md

```diff
@@ -101,30 +101,37 @@ end
 ```
 
 ## Captcha-Support
+
 The captcha support depends on [EasyCaptcha](https://github.com/phatworx/easy_captcha). See further documentation there.
 
 ### Installation
 
 1. Add EasyCaptcha to your `Gemfile` with
+
 ```ruby
 gem 'easy_captcha'
 ```
-2. Run the initializer
+
+1. Run the initializer
+
 ```ruby
 rails generate easy_captcha:install
 ```
-3. Enable captcha - see "Configuration" of Devise Security above.
-4. Add the captcha in the generated devise views for each controller you have activated
+
+1. Enable captcha - see "Configuration" of Devise Security above.
+1. Add the captcha in the generated devise views for each controller you have activated
+
 ```erb
 <p><%= captcha_tag %></p>
 <p><%= text_field_tag :captcha %></p>
 ```
 
 ## Schema
 
-Note: Unlike Devise, devise-security does not currently support mongoid.  Pull requests are welcome!
+Note: Unlike Devise, devise-security does not currently support mongoid. Pull requests are welcome!
```

Comment:
I'd like to remove the `Pull requests are welcome!` given that we've discussed on an issue recently about that being pretty complicated.

Reply:
This is just a reformatting, if we want to remove that line, I suggest a new PR.